app/signals/apps/classification/models/training_set.py

```diff
@@ -0,0 +1,39 @@
+from django.contrib.gis.db import models
+
+from signals import settings
+from signals.apps.services.domain.checker_factories import ContentCheckerFactory
+from signals.apps.services.domain.mimetypes import MimeTypeFromContentResolverFactory, \
+    MimeTypeFromFilenameResolverFactory
+from signals.apps.services.validator.file import MimeTypeAllowedValidator, MimeTypeIntegrityValidator, \
+    ContentIntegrityValidator, FileSizeValidator
+
+
+class TrainingSet(models.Model):
+    """
+    This model represents a training set consisting of a single XLSX file, with a "Main", "Sub" and "Text" column
+    """
+    created_at = models.DateTimeField(editable=False, auto_now_add=True)
+
+    name = models.CharField(max_length=255, null=False, blank=False)
+
+    file = models.FileField(
```

Comment:
Hier moeten we nog even de DWH storage gebruiken, in plaats van de media storage. De media storage is namelijk publiek bereikbaar. Dat is niet zo wenselijk voor het ML model.